handler/handler.go

```diff
@@ -0,0 +1,50 @@
+package handler
+
+import (
+	"log"
+	"net/http"
+	"strconv"
+
+	embed "github.com/Clinet/discordgo-embed"
+	"github.com/bwmarrin/discordgo"
+	env "github.com/infiniteloopcloud/discord-downdetector/env"
+)
+
+const (
+	warning = 0xD10000
+)
+
+var channelName string
+
+// Check is the endpoint alive
+func Handle(body env.Check) (string, *discordgo.MessageEmbed, error) {
+	code := checkHealth(body)
+	if code != 200 {
```

Comment:
See changes above. Also in Go we don't use numbers for these kind of checks, we use http.StatusOk.